tests/test_input_test.py

```diff
@@ -0,0 +1,12 @@
+from pydicer.input.test import TestInput
+import os
+
+def test_valid_input_path():
```

Comment:
I don't think docstrings are required for tests right? They should usually be pretty obvious what they are doing. We might have have to add the appropriate 'pylint disable to our test files though so that this won't flag.

Reply:
Yes I'll add the disable option for the linter and I believe we don't need more than a brief comment describing the test